docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -38,9 +38,10 @@ Freshness checks use an asset's latest update time to determine if it's overdue.
 
 Before continuing, you should be familiar with:
 
-- [Software-defined Assets](/concepts/assets/software-defined-assets)
+- [Asset definitions](/concepts/assets/software-defined-assets)
```

Comment:
Should this change?

Reply:
"Asset definitions" is the preferred nomenclature in these kinds of docs now